en/docs/chapter_sorting/counting_sort.md

```diff
@@ -71,14 +71,14 @@ The implementation code of counting sort is shown below:
 
 ## Algorithm characteristics
 
-- **Time complexity is $O(n + m)$, non-adaptive sort**: Involves traversing `nums` and `counter`, both using linear time. Generally, $n \gg m$, and the time complexity tends towards $O(n)$.
+- **Time complexity is $O(n + m)$, non-adaptive sort**: involves traversing `nums` and `counter`, both using linear time. Generally, $n \gg m$, and the time complexity tends towards $O(n)$.
```

Comment:
Should we capitalize the first letter of `involves`?

Reply:
Yes! And this sentence has grammar error. "involves" => "It involves".